src/main/java/com/cburch/logisim/prefs/AppPreferences.java

```diff
@@ -115,10 +114,7 @@ public void localeChanged() {
     public void preferenceChange(PreferenceChangeEvent event) {
       final var prefs = event.getNode();
       final var prop = event.getKey();
-      if (ACCENTS_REPLACE.getIdentifier().equals(prop)) {
-        getPrefs();
-        LocaleManager.setReplaceAccents(ACCENTS_REPLACE.getBoolean());
-      } else if (prop.equals(TEMPLATE_TYPE)) {
+      if (prop.equals(TEMPLATE_TYPE)) {
```

Comment:
`TEMPLATE_TYPE.equals(prop)` please to mitigate potential NPE

Reply:
As I just removed, and do not want to mangle refactoring with this PR I will leave it as is.